config/newrelic.yml

```diff
@@ -1,229 +1,694 @@
+  # audit_log.path: "/audit_log"
+
+  # Specify a list of constants that should prevent the agent from starting
+  # automatically. Separate individual constants with a comma ,.
+  # For example, Rails::Console,UninstrumentedBackgroundJob.
+  # autostart.denylisted_constants: "rails::console"
+
+  # Defines a comma-delimited list of executables that the agent should not
+  # instrument. For example, rake,my_ruby_script.rb.
+  # autostart.denylisted_executables: "irb,rspec"
+
+  # Defines a comma-delimited list of Rake tasks that the agent should not
+  # instrument. For example, assets:precompile,db:migrate.
+  # autostart.denylisted_rake_tasks: "about,assets:clean,assets:clobber,assets:environment,assets:precompile,assets:precompile:all,db:create,db:drop,db:fixtures:load,db:migrate,db:migrate:status,db:rollback,db:schema:cache:clear,db:schema:cache:dump,db:schema:dump,db:schema:load,db:seed,db:setup,db:structure:dump,db:version,doc:app,log:clear,middleware,notes,notes:custom,rails:template,rails:update,routes,secret,spec,spec:features,spec:requests,spec:controllers,spec:helpers,spec:models,spec:views,spec:routing,spec:rcov,stats,test,test:all,test:all:db,test:recent,test:single,test:uncommitted,time:zones:all,tmp:clear,tmp:create,webpacker:compile"
+
+  # Backports the faster Active Record connection lookup introduced in Rails 6,
+  # which improves agent performance when instrumenting Active Record. Note that
+  # this setting may not be compatible with other gems that patch Active Record.
+  backport_fast_active_record_connection_lookup: true
+
+  # If true, the agent captures attributes from browser monitoring.
+  # browser_monitoring.attributes.enabled: false
+
+  # Prefix of attributes to exclude from browser monitoring. Allows * as wildcard
+  # at end.
+  # browser_monitoring.attributes.exclude: []
+
+  # Prefix of attributes to include in browser monitoring. Allows * as wildcard at
+  # end.
+  # browser_monitoring.attributes.include: []
+
+  # This is true by default, this enables auto-injection of the JavaScript header
+  # for page load timing (sometimes referred to as real user monitoring or RUM).
+  browser_monitoring.auto_instrument: false
+
+  # Manual override for the path to your local CA bundle. This CA bundle will be
+  # used to validate the SSL certificate presented by New Relic's data collection
+  # service.
+  # ca_bundle_path: nil
+
+  # Enable or disable the capture of memcache keys from transaction traces.
+  # capture_memcache_keys: false
+
+  # When true, the agent captures HTTP request parameters and attaches them to
+  # transaction traces, traced errors, and TransactionError events. When using the
+  # capture_params setting, the Ruby agent will not attempt to filter secret
+  # information. Recommendation: To filter secret information from request
+  # parameters,use the attributes.include setting instead. For more information,
+  # see the Ruby attribute examples.
+  # capture_params: false
```

Comment:
`capture_params` was set to `false` in the previous configuration. Is there a reason this is commented out?